service/controller/key/common.go

```diff
@@ -278,7 +282,7 @@ func StackNameTCCPI(getter LabelsGetter) string {
 }
 
 func StackNameTCCPN(getter LabelsGetter) string {
-	return fmt.Sprintf("cluster-%s-tccpn-%s", ClusterID(getter), ControlPlaneID(getter))
+	return fmt.Sprintf("cluster-%s-tccpn", ClusterID(getter))
```

Comment:
We cannot change the tccpn stack name  like this, it was already configured with some name and this caused a new duplicate stack to be create when upgrading from old version.

So reverting to previous state